CAIRN Info.js

```diff
@@ -84,6 +84,22 @@ function doWeb(doc,url)
 						tags.push(keywords[i])
 					}
 				}
+				
+				// The default value for PDF download is on an HTML page that calls the actual download
+				// we need to add this attachment after the import translator has been running
+				// We remove the attached PDF because they won't be working
+				for(var i=0, n=item.attachments.length; i<n; i++) {
```

Comment:
@adam3smith I must admin that I have copied/pasted what is done [in Embedded metadata.js](https://github.com/zotero/translators/blob/master/Embedded%20Metadata.js#L480-L484) without thinking about it, I will update that tonight yes, thanks for the input.